packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1288,7 +1304,7 @@ class SelectionOverlay {
   // Manages the context menu. Not necessarily visible when non-null.
   final ContextMenuController _contextMenuController = ContextMenuController();
 
-  bool get _contextMenuControllerIsShown => _contextMenuController.isShown;
```

Comment:
What were the side effects of using the same `ContextMenuController` for both the text selection toolbar and the spell check suggestions toolbar?